pkg/kube/client.go

```diff
@@ -579,8 +579,8 @@ func batchPerform(infos ResourceList, fn func(*resource.Info) error, errs chan<-
 		}
 		wg.Add(1)
 		go func(i *resource.Info) {
+			defer wg.Done()
 			errs <- fn(i)
-			wg.Done()
```

Comment:
why you make this change here ? It's seems the same effect.

Reply:
This change is meaningless, but in general it's a good practice to do `wg.Done()` by `defer` so never forget to do that in any condition

Reply:
I reverted this change